src/h5web/visualizations/line/LineVis.tsx

```diff
@@ -27,7 +28,7 @@ function LineVis(): JSX.Element {
     <div className={styles.root}>
       <VisCanvas
         abscissaConfig={{
-          indexDomain: extendDomain([0, data.length - 1], 0.01),
+          indexDomain: extendDomain([0, values.size - 1], 0.01),
```

Comment:
As `values` is 1D, `size` is equal to the length of the flattened array but shorter to write.